src/components/HdTagsList.vue

```diff
@@ -23,18 +32,36 @@ export default {
 @import 'homeday-blocks/src/styles/mixins.scss';
 
 .tags-list {
+  $root: &;
   display: flex;
   flex-wrap: wrap;
   min-height: #{$sp-l + $sp-s};
   margin-bottom: $sp-m;
+
   &__tag {
     display: flex;
     align-items: center;
-    height: $sp-l;
-    background-color: getShade($quaternary-color, 50);
-    border-radius: 3px;
     padding: 0 $sp-s;
     margin: $sp-s $sp-s 0 0;
   }
+
+  &--primary {
+    #{$root}__tag {
+      height: $sp-l;
+      background-color: getShade($quaternary-color, 50);
+      border-radius: 3px;
+    }
+  }
+  &--secondary {
+    #{$root}__tag {
+      height: $sp-m;
+      background-color: $cello-blue;
+      border-radius: 12px;
+      color: $white;
+      font-weight: 600;
+      font-size: 10px;
+      line-height: 17px;
```

Comment:
This is probably meant to be `18px` otherwise the text won't be vertically centered.
```suggestion
      line-height: 18px;
```